tests/console/journalctl.pm

```diff
@@ -37,6 +37,11 @@ sub check_journal {
     return script_run("if [ -s $filename ]; then true; else false; fi");
 }
 
+sub skip_fss_check {
+    # FSS check is not supported by SLE. Disable this check for SLES and Leap
+    return is_sle || is_leap;
```

Comment:
The comments in the [bsc#1171858#c4](https://bugzilla.suse.com/show_bug.cgi?id=1171858#c4) suggest to leave the test for Leap, why is here to be removed?

Reply:
Good catch, I missed that one. Putting Leap back on.